udf-compiler/src/test/scala/com/nvidia/spark/OpcodeSuite.scala

```diff
@@ -35,6 +35,7 @@ class OpcodeSuite extends FunSuite {
 
   val conf: SparkConf = new SparkConf()
     .set("spark.sql.extensions", "com.nvidia.spark.udf.Plugin")
+    .set("spark.rapids.sql.test.enabled", "true")
```

Comment:
@wjxiz1992 Is this needed?  It makes all the tests that are supposed to fall back to CPU fail.

Reply:
I will remove this after the feature is implemented. It's enabled now to see the actual error message like `Unsupported instruction: 182`.

Reply:
I see.  Okay.